client/src/components/material.jsx

```diff
@@ -28,13 +10,13 @@ const Blob = Styled.div`
   height: 150px;
   background: url("https://www.patagonia.com/dw/image/v2/bdjb_PRD/on/demandware.static/-/Library-Sites-PatagoniaShared/default/dw6fbb2428/fpc/programs/brewer_c_0001_cc_WEB-1200x1200.jpg?q=60&amp;sw=600&amp");
   background-size: cover;
-  animation: animate 30s linear infinite;
+  animation: animate 20s linear infinite;
   @keyframes animate {
     0%, 100% {
-    border-radius: 64% 36% 27% 73% / 55% 58% 42% 45%;
+    border-radius: 44% 36% 27% 73% / 55% 58% 42% 45%;
     }
     25% {
-      border-radius: 73% 27% 26% 38% / 73% 36% 64% 27%
+      border-radius: 63% 37% 26% 38% / 73% 36% 64% 27%
```

Comment:
@gigicupp Amazing job figuring out the blob and animation for it! So impressive!